external/fv3fit/fv3fit/keras/_models/recurrent.py

```diff
@@ -546,6 +549,17 @@ class StepwiseModel(PureKerasModel):
     include air_temperature and specific_humidity
     """
 
+    def __init__(
+        self,
+        input_variables: Iterable[Hashable],
+        output_variables: Iterable[Hashable],
+        output_metadata: Iterable[Dict[str, Any]],
+        model: tf.keras.Model,
+        sample_dim_name: str,
```

Comment:
This is the one model that still has sample dim as an arg, since it looks like its method to integrate forward in time works on a dataset that can have any name for its sample dim.

Reply:
This is fine.